Apps/Playground/ios/Playground.xcodeproj/project.pbxproj

```diff
@@ -634,6 +634,46 @@
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Playground/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				LIBRARY_SEARCH_PATHS = (
```

Comment:
What's all this? Something XCode 13 did on its own?

Reply:
Yes I believe this is because before this field was auto-populated by app template. Now that we have added a custom path everything gets dumped to the pbxproj file.